### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by JosephGJ**
ref: #1434 

Documentation updates across the whole project. Primarily focused on `/docs` and `/project`. Focusing on `/docs` as it is more aimed at the user, where as `/project` is more focused on contributors and maintainers. 

Individual files that still need to be completed in `/docs`:
- [x] `README.md`
- [ ] `cluster.md`
- [x] `config.md`
- [x] `logs.md`
- [x] `org.md`
- [x] `password.md`
- [x] `service.md`
- [x] `stack.md`
- [x] `stats.md`
- [x] `team.md`
- [x] `user.md`
- [x] `version.md`

When I complete these, I'll move onto the main `README.md` and then `/project`.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,115 +1,346 @@
...
 
- * [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
- * [Docker for Windows](https://docs.docker.com/docker-for-windows/install/)
- * [Docker for Linux](https://docs.docker.com/engine/installation/) (see specific distribution under the sidebar).
- 
-#### Linux Node:
+### Prerequisites
 
-If you're running on Linux, there are a few steps you must perform for your system in addition to installing Docker.
+The only prerequisite required to run `amp` is Docker 17.03 or greater.
```
Comment (bquenin) [inline docs/README.md]: should we go ahead and recommend 17.06?

File: docs/README.md [hunk 2]
```diff
@@ -1,115 +1,346 @@
...
+`go` or `gnu` on your system, just docker.
 
-> *Or, if you prefer, fork this repo and build your own CLI. The entire toolchain
-has been containerized, so all you need on your system is Docker 17.03 or greater. To
-build the CLI, add the appropriate path under the `./bin` directory to your shell path.
-Then run `ampmake buildall-cli`; this will build cross-compiled versions of the cli
-(`amp`) and place them in the appropriate locations under `./bin`. (As a special case,
-`make build-cli` can be used to build your OS-specific executable, if you already
-have gnu make installed on your system.)*
+To build the CLI, add the appropriate path under the `./bin` directory to your shell path, then run:
```
Comment (bquenin) [inline docs/README.md]: Maybe we could enhance the example with:
```
$ cd $GOPATH/src
$ mkdir -p github.com/appcelerator
$ cd github.com/appcelerator
$ git clone https://github.com/appcelerator/amp
$ cd amp
$ export PATH=bin/linux/amd64:$PATH
```

File: docs/logs.md [hunk 3]
```diff
@@ -1,55 +1,58 @@
...
+Options:
+      --container string   Filter by the given container
+  -f, --follow             Follow log output
+  -h, --help               Print usage
+  -i, --include            Include AMP logs
+  -k, --insecure           Control whether amp verifies the server's certificate chain and host name
+  -m, --meta               Display entry metadata
+      --msg string         Filter the message content by the given pattern
+      --node string        Filter by the given node
+  -n, --number int         Number of results (default 1000)
```
Comment (bquenin) [inline docs/logs.md:23]: missing this option:
```
  -r, --raw                Display raw logs (no prefix)
```

**Review by bquenin — COMMENTED** [review]
This looks awesome! Great job on the documentation @JosephGJ ! :+1:

File: docs/README.md [hunk 4]
```diff
@@ -1,115 +1,346 @@
...
- 
-#### Linux Node:
+### Prerequisites
 
-If you're running on Linux, there are a few steps you must perform for your system in addition to installing Docker.
+The only prerequisite required to run `amp` is Docker 17.03 or greater.
 
-Check if you have IPVS kernel modules loaded:
+We recommend installing the Docker CE stable channel release 17.06 on your system.
+Make sure you check the "What to know before you install" section to ensure your system meets the requirements.
```
Comment (bquenin) [inline docs/README.md]: Does the `What to know before you install` section actually exists? I couldn't find it.

File: docs/README.md (hunk 1 above)
Comment (JosephGJ) [reply docs/README.md]: I have said this on line 29, 2 lines below this. I can move that before this statement if you like?

File: docs/README.md (hunk 4 above)
Comment (JosephGJ) [reply docs/README.md]: The `Docker for Linux` installation doesn't have a `What to know before you install` section. There is on both the macOS and Windows links. There is also a section in the sidebar called `Post-installation steps for Linux` which we could point the users towards?

File: docs/README.md (hunk 2 above)
Comment (JosephGJ) [reply docs/README.md]: done

File: docs/logs.md (hunk 3 above)
Comment (JosephGJ) [reply docs/logs.md:23]: fixed

File: docs/README.md (hunk 1 above)
Comment (bquenin) [reply docs/README.md]: Yeah I think you can get rid of the 2 lines below and go ahead an recommend 17.06 altogether.

File: docs/README.md (hunk 4 above)
Comment (bquenin) [reply docs/README.md]: Oh sorry, I though it was referring to a section in our docs. Got it.

Comment (JosephGJ) [reply docs/README.md]: I made it a bit more explicit as to where the `what to know before you install` section is.

File: docs/README.md (hunk 1 above)
Comment (JosephGJ) [reply docs/README.md]: Done that.

File: docs/README.md [hunk 5]
```diff
@@ -1,115 +1,364 @@
...
-    #
-    # if necessary, load IPVS kernel modules (ip_vs, ip_vs_rr) for swarm mode networking and load-balancing
-    # Note: these modules are needed to run a local swarm using Docker-in-Docker on the host,
-    # but this step is not necessary if the host is already a member of a swarm.
-    # Note: loading ip_vs_rr will also ensure ip_vs is loaded.
-    $ sudo modprobe ip_vs_rr
+#### MacOS
 
-Increase virtual memory (needed for Elasticsearch):
+On macOS, Docker can be installed via the Docker website.
```
Comment (neha-viswanathan) [inline docs/README.md]: Fix typo - `macOS` to `MacOS`

Comment (JosephGJ) [reply docs/README.md]: fixed

**Review by ndegory — APPROVED** [review]

**Pull request merged**